lib/jekyll.rb

```diff
@@ -32,6 +32,9 @@ def require_all(path)
 require "colorator"
 
 SafeYAML::OPTIONS[:suppress_warnings] = true
+ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
```

Comment:
`Parallel.processor_count` does the same thing, it's cross-platform though. You're using `parallel` anyway, so it makes sense to use it?

https://github.com/grosser/parallel/blob/master/lib/parallel/processor_count.rb